web/packages/core/src/ruffle-player.ts

```diff
@@ -1071,6 +1077,17 @@ export class RufflePlayer extends HTMLElement {
                     <li><a href="#" id="panic-view-details">View Error Details</a></li>
                 `;
                 break;
+            case PanicError.WasmDownload:
+                // Usually a transient network error or botched deployment
+                errorBody = `
+                    <p>Ruffle has encountered a major issue whilst trying to initialize.</p>
+                    <p>This can often resolve itself, so you can try reloading the page.</p>
+                    <p>Otherwise, there may be a server-side issue, so contact the server administrator.</p>
```

Comment:
"please contact the website administrator" sounds a bit better for non-technical users

Reply:
"there may be a server-side issue, so please contact the website administrator" or just "please contact the website administrator"? I did the first, but I'll do the second and squash the commits if that is better.